lib/ext2fs/Makefile.in

```diff
@@ -154,6 +156,8 @@ SRCS= ext2_err.c \
 	$(srcdir)/closefs.c \
 	$(srcdir)/crc16.c \
 	$(srcdir)/crc32c.c \
+	$(srcdir)/crc32c_intel.c \
+	$(srcdir)/crc32c_intel_pcl.S \
```

Comment:
This assumes that the makefile rules have a .S -> .o rule, and that compilers will accept the .S as a assembly file as input.   That's not very portable.  There are people who build e2fsprogs on NetBSD, FreeBSD, MacOS, Windows, Android, OpenSolaris, AIX, etc.   There seems to be an unfortunate assumption that "all the world's x86" and "all the world's Linux", or "the only compiler is gcc" or "the only compiler is clang".